src/xrEngine/imgui_tools.cpp

```diff
@@ -0,0 +1,16 @@
+#include "stdafx.h"
+#include <imgui.h>
+#include "../xrServerEntities/script_engine_debugger.h"
+
+void InitImguiTools()
+{
+    auto& States = Engine.External.EditorStates;
+    bool* luaDebug = &States[static_cast<u8>(EditorUI::LuaDebug)];
+    if (ImGui::MenuItem("Lua: Attach to VSCode (Doesn't work)", nullptr, luaDebug))
+    {
+        if (*luaDebug)
+        {
+            //AttachDebugger(); //Getting crash in ljtab.c after executing this function: Unhandled exception thrown: read access violation.
```

Comment:
Wasn't able to make this work, it goes deep into namespaces and lua source files and it's not clear at all what to look for

Reply:
![image](https://github.com/user-attachments/assets/7263d0c8-f5c0-4219-89d9-7b5b6844bb84)

Reply:
![image](https://github.com/user-attachments/assets/a692292c-db5d-4490-92a2-b0f4b5b664f2)